content/root/glossary.textile

```diff
@@ -0,0 +1,212 @@
+
+See "Wikipedia":https://en.wikipedia.org/wiki/MQTT for further information.
+
+h2(#mqtt-adapter). MQTT Adapter
+
+The Ably MQTT Adapter can translate between MQTT and the Ably protocol.
+
+See the "MQTT Adapter documentation":https://ably.com/documentation/mqtt for further information.
+
+h2(#presence). Presence
+
+Presence enables clients in a channel to be aware of other presence members in the same channel. Each client has a presence state that is updated as a presence event when it changes, such as when they join or leave the channel.
+
+See the "presence documentation":/realtime/presence for further information.
+
+h2(#pub-sub). Pub/Sub
+
+Publish-subscribe (or pub/sub), using Ably terms, is a pattern of information exchange where a publisher does not need to interact directly with, or even know the existence or identities of, any subscribers. Conversely, a subscriber can gain access to published information without interacting directly with the publisher. The ratio of publishers to subscribers can be M:N, across any number of channels.
+
+h2(#publisher). Publisher
+
+A publisher is an entity that sends messages on one or more channels.
+
+h2(#push-notification). Push notifications
+
+Push notifications are messages that are pushed to a device's user interface such as a visual notification. Ably can deliver push notifications using platforms such as Apple's Push Notification service or Google's Firebase Cloud Messaging service.
+
+See the "push notification documentation":/realtime/push for further information.
+
+h2(#queues). Queues
+
+Reactor Queues provide a service that distributes messages between subscribers, so that each message is only received by one subscriber. This model is referred to as "competing consumers" and is different to the pub/sub model where each message is delivered to every subscriber. Ably uses integration rules to publish messages received in a channel into a queue.
+
+See the "queues documentation":/general/queues for further information.
+
+h2(#realtime-sdk). Realtime client library SDK
+
+The realtime client library SDKs are used to add long-lived realtime connections to Ably within applications.
+
+See the "SDK downloads page":https://ably.com/download for further information on SDKs and the available languages.
+
+h2(#rest-sdk). REST client library SDK
+
+The REST client library SDKs provide an API to interact with the Ably REST (HTTP) interface.
+
+See the "SDK downloads page":https://ably.com/download for further information on SDKs and the available languages.
+
+h2(#rewind). Rewind
+
+Rewind is a feature that enables a client to subscribe to a channel from a position earlier than the current point in time. The @rewind@ parameter can specify an earlier position as a time interval or set number of messages. When a client first attaches to a channel using rewind, all messages from that earlier position are replayed to them.
```

Comment:
"The @rewind@ parameter" - connection or channel parameter?

Reply:
Reworded to state it's a channel parameter.